pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.554.3</version>
+        <version>1.596.1</version>
```

Comment:
@jglick This plugins requires `analysis-core` plugin and this requieres `1.596.1`